scrapers/idsForArcGIS.mjs

```diff
@@ -0,0 +1,39 @@
+import { dataset } from '../util/Dataset.mjs';
+
+export default ({ database, DataScraper }) => {
+	return DataScraper(database, 'idsForArcGIS', async () => {
+        let results = [];
+
+		const ids = [//'b70198178ff44462b00c5b5c0012668d_0', //ArlingtonVa currently broken
+        '8cae9e82135d4bd0925f8c5e71254924_0', //AtlantaGa
```

Comment:
Make sure this builds with the app. This whitespace will not make our liner happy. To test it, checkout the app, update the submodule to this branch and see what the build system has to say.

Reply:
My VScode had tabs set up wrong. Fixed and changed!